README.md

```diff
@@ -65,34 +65,34 @@ The [docs](https://auto-dl.readthedocs.io/en/latest/index.html) are the perfect
 
 * You can directly jump to the [user guide](https://docs.auto-dl.tech/en/latest/userguide.html) :)
 
-## Developer Guidelines
+## Developer Guidelines 👩‍💻 
 
 The [contributing guidelines](https://github.com/Auto-DL/Auto-DL/blob/v1-beta/CONTRIBUTING.md) hosts all the information about building Auto-DL from source, how to contribute code and documentation, who to contact about what, etc.
 
-## Get started with Auto-DL (For Developers)
+## Get started with Auto-DL (For Developers) 🌐
 
 Setup Auto-DL:
 
-- [Bare-metal way](https://docs.auto-dl.tech/en/latest/installing.html#setup-from-source-code)
-- [Using Docker](https://docs.auto-dl.tech/en/latest/installing.html#setup-using-docker)
+- [Bare-metal way](https://docs.auto-dl.tech/en/latest/installing.html#setup-from-source-code) 👷
+- [Using Docker](https://docs.auto-dl.tech/en/latest/installing.html#setup-using-docker)  🐳
 
 ---
 
-### Features/Enhancements planned
+### Features/Enhancements planned 🗺
```

Comment:
```suggestion
### Features/Enhancements planned 🗺
```
The emoji (map) and the rendered (world map) form are different, I'd strongly suggest changing it to something more suitable...